websites/D/Discords.com/presence.ts

```diff
@@ -195,14 +195,60 @@ presence.on("UpdateData", async () => {
         url: document.location.href
       }
     ];
+    // discord.bio
+  } else if (document.location.pathname.includes("/profiles")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.smallImageKey = "reading";
+    presenceData.smallImageText = "Browsing...";
+    presenceData.details = "Browsing top bios";
+  } else if (document.location.pathname === "/bio/premium") {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.details = "Viewing 💎 premium plans";
+    presenceData.buttons = [
+      {
+        label: "View Premium Plans",
+        url: document.location.href
+      }
+    ];
+  } else if (document.location.pathname.includes("/customise")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.details = "Editing bio";
+  } else if (document.location.pathname.includes("/settings")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.details = "Viewing ⚙️ settings";
+  } else if (document.location.pathname.includes("/p/")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    const profileName = document.querySelector(
+        "span.text-white.font-bold.text-2xl"
+      )?.textContent,
+      premium = document.querySelector(
+        "div.text-xs.uppercase.tracking-widest.font-bold.text-blue-300"
+      )?.textContent;
+    presenceData.details = `Viewing ${
+      profileName
+        ? profileName.endsWith("s")
+          ? `${`${profileName}'`}`
+          : `${`${profileName}'s`}`
+        : "Unknown"
+    } bio`;
+    presenceData.state = `${premium ? "💎 Premium User" : "🎟️ Normal User"}`;
+    presenceData.buttons = [
+      {
+        label: "View Bio",
+        url: document.location.href
+      }
+    ];
+  } else if (document.location.pathname.includes("/bio")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.details = "Viewing home page";
   }
+
   if (!showButtons) delete presenceData.buttons;
   if (showTimestamp) presenceData.startTimestamp = browsingStamp;
 
-  if (presenceData.details == null) {
+  if (presenceData.details === null) {
     presence.setTrayTitle();
     presence.setActivity();
-  } else {
-    presence.setActivity(presenceData);
   }
+  presence.setActivity(presenceData);
```

Comment:
What is the point of this statement then

Reply:
![image](https://user-images.githubusercontent.com/9348108/120345796-18961180-c2fb-11eb-82c6-743b3ee73a3f.png)

Reply:
lol